.github/workflows/checks.yml

```
@@ -282,7 +272,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ "3.8", "3.9", "3.10", "3.11" ]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12" ]
```

nit:
```suggestion
        python-version: [ "3.8", "3.9", "3.10", "3.11", "3.12" ]
```
It can be found also in a few other places.